staging/src/k8s.io/apiserver/pkg/admission/plugin/webhook/mutating/dispatcher.go

```diff
@@ -97,6 +97,13 @@ func (a *mutatingDispatcher) callAttrMutatingHook(ctx context.Context, h *v1beta
 		return &webhookerrors.ErrCallingWebhook{WebhookName: h.Name, Reason: fmt.Errorf("Webhook response was absent")}
 	}
 
+	for k, v := range response.Response.Annotations {
+		if err := attr.AddAnnotation(h.Name+"/"+k, v); err != nil {
+			return &webhookerrors.ErrCallingWebhook{WebhookName: h.Name,
```

Comment:
I'm not sure this should be a fatal error... the in-tree annotation error handling just logs a warning